packages/core/src/browser/shell/shell-layout-restorer.ts

```diff
@@ -71,8 +71,7 @@ export class ShellLayoutRestorer implements CommandContribution {
             const serializedLayoutData = await this.storageService.getData<string>(this.storageKey);
             if (serializedLayoutData !== undefined) {
                 const layoutData = await this.inflate(serializedLayoutData);
-                app.shell.setLayoutData(layoutData);
-                await app.shell.pendingUpdates;
```

Comment:
Do we still need `pendingUpdated` as the public API on the shell or it can be encapsulated in the shell?